modules/developers-guide/pages/cli-reference/dfx-canister.adoc

```diff
@@ -64,14 +64,16 @@ For example, to register unique canister identifiers for a project when the {IC}
 dfx canister create --all
```

Comment:
L55: Overriding the default network => Overriding the default deployment option

L57: on the local network => on the local canister execution environment